src/tools/perf/perftest_params.c

```diff
@@ -583,6 +587,16 @@ ucs_status_t parse_opts(struct perftest_context *ctx, int mpi_initialized,
                 return status;
             }
             break;
+        case 'K':
+            str = strtok(optarg, ":");
+            if (!str) {
+                ucs_error("Invalid Infiniband source interface specified");
+                return UCS_ERR_INVALID_PARAM;
+            }
+            ctx->ib.ca      = strdup(str);
+            str             = strtok(NULL, ":");
+            ctx->ib.ca_port = str ? atoi(str) : 1;
+            break;
```

Comment:
maybe we need to fail if several RTEs are requested? E.g. MPI is initialized and `-K` specified

Reply:
added mpi/ib-mad exclusion